### PR TITLE
Add additional feed to avoid random restore errors

### PR DESCRIPTION
**Pull request by ViktorHofer**
in official builds.

Fixes https://github.com/dotnet/corefx/issues/36382

### CODE REVIEW

File: eng/pipelines/corefx-base.yml [hunk 1]
```diff
@@ -138,7 +138,7 @@ jobs:
                 nugetConfigPath: 'NuGet.config'
                 restoreDirectory: '$(Build.SourcesDirectory)\.packages'
               env:
-                VSS_NUGET_EXTERNAL_FEED_ENDPOINTS: '{"endpointCredentials": [{"endpoint":"https://devdiv.pkgs.visualstudio.com/_packaging/8f470c7e-ac49-4afe-a6ee-cf784e438b93/nuget/v3/index.json", "username":"dn-bot","password":"$(dn-bot-devdiv-nuget-feed-read)"}]}'
+                VSS_NUGET_EXTERNAL_FEED_ENDPOINTS: '{"endpointCredentials": [{"endpoint":"https://devdiv.pkgs.visualstudio.com/_packaging/8f470c7e-ac49-4afe-a6ee-cf784e438b93/nuget/v3/index.json", "username":"dn-bot","password":"$(dn-bot-devdiv-nuget-feed-read)"},{"endpoint":"https://devdiv.pkgs.visualstudio.com/_packaging/VS/nuget/v3/index.json", "username":"dn-bot","password":"$(dn-bot-devdiv-nuget-feed-read)"}]}'
```
Comment (ericstj) [inline eng/pipelines/corefx-base.yml]: nit: can this be split on new lines?

Comment (ViktorHofer) [reply eng/pipelines/corefx-base.yml]: To be honest I don't know. @safern?

Comment (safern) [reply eng/pipelines/corefx-base.yml]: Technically yes. I haven't tried for quoted stuff, but it should work due to the yaml syntax model. I would give it a try.

**Comment by safern** [comment]
Make sure to test with an official build before merging to not have to submit a follow up PR if something breaks.

**Comment by ViktorHofer** [comment]
https://dev.azure.com/dnceng/internal/_build/results?buildId=142058

**Comment by ViktorHofer** [comment]
https://dev.azure.com/dnceng/internal/_build/results?buildId=142328

**Comment by ViktorHofer** [comment]
Added the remaining missing feeds. Official build is finally passing again :)

**Comment by ericstj** [comment]
Any idea what those additional feeds were?  The trick I used before to reverse the GUID to a named feed didn't work for the 97a41293.. one you added 😕

**Comment by ViktorHofer** [comment]
No I'm not sure. But it could be that arcade added them. I used the logs which printed these feeds.

**Comment by ViktorHofer** [comment]
I expect all of this to be temporary as we really should migrate to a non-feed solution with all internal tools living in dnceng.

**Pull request merged**